packages/designsystem/src/stories/profile/UpdateUsernameForm.story.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <Story title="Profile/UpdateUsernameForm Form" icon="bi:person-fill">
+    <Variant title="default">
+      <UpdateUsernameForm
+        :user="{
+          id: 1,
+          email: 'contact@locokit.com',
+          profile: 'ADMIN',
+          name: 'Companion Cube',
+        }"
+        @submit="logEvent('submit', $event)"
+      />
+    </Variant>
+  </Story>
+</template>
+
+<script setup lang="ts">
+import { logEvent } from 'histoire/client'
+
+import UpdateUsernameForm from '../../components/profile/UpdateUsernameForm/UpdateUsernameForm.vue'
+</script>
+
+<docs lang="md">
+### UpdateUsernameForm
+
+Form which allow an user to update its name/username if this user is already logged.
```

Comment:
```suggestion
Form allowing a user to update its firstname, lastname and username.
```

Reply:
By the way, I don't find the `UpdateUsernameForm` component ? Could you point me the file ?

Reply:
I already fix in https://github.com/locokit/locokit/pull/205/commits/f8712c7c7bbb84e847d10c62ee49f0a29d678dd0